packages/lab/stories/static-list/static-list.stories.tsx

```diff
@@ -0,0 +1,149 @@
+import { CalendarIcon, OverflowMenuIcon, VideoIcon } from "@salt-ds/icons";
+import {
+  StaticList,
+  StaticListItem,
+  StaticListItemContent,
+  type StaticListProps,
+} from "@salt-ds/lab";
+import type { Meta, StoryFn } from "@storybook/react";
+import React, { Fragment, useState } from "react";
+import { complexEventsData, eventsData } from "../assets/exampleData";
+
+export default {
+  title: "Lab/Static List",
+  component: StaticList,
+} as Meta<typeof StaticList>;
+
+export const Default: StoryFn<StaticListProps> = () => {
+  const [listCount, setListCount] = useState(3);
+
+  const handleListItem = () => {
+    setListCount((prev) => prev + 1);
+  };
+  const handleReset = () => {
+    setListCount(3);
+  };
+  return (
+    <StackLayout>
+      <StackLayout direction={"row"}>
+        <Button
+          onClick={handleListItem}
+          disabled={listCount > eventsData.length - 1}
+        >
+          Show next event
+        </Button>
+        <Button onClick={handleReset}>Reset</Button>
+      </StackLayout>
+      <StaticList style={{ width: "320px", height: "250px" }}>
+        {eventsData.slice(0, listCount).map((event, _index) => (
+          <StaticListItem key={event}>
+            <StaticListItemContent>
+              <Text>{event}</Text>
+            </StaticListItemContent>
+          </StaticListItem>
+        ))}
+      </StaticList>
+    </StackLayout>
+  );
+};
+
+export const ComplexLabel: StoryFn<StaticListProps> = () => {
+  return (
+    <StaticList style={{ width: "320px" }}>
+      {complexEventsData.map(({ title, time }) => (
+        <StaticListItem key={title}>
+          <StaticListItemContent>
+            <StackLayout gap={0.5}>
+              <Text color="inherit">{title}</Text>
+              <Text styleAs="label" color="secondary">
+                {time}
+              </Text>
+            </StackLayout>
+          </StaticListItemContent>
+        </StaticListItem>
+      ))}
+    </StaticList>
+  );
+};
+
+export const WithIcons: StoryFn<StaticListProps> = () => {
+  return (
+    <StaticList style={{ width: "320px" }}>
+      {complexEventsData.map(({ title, time }) => (
+        <StaticListItem key={title}>
+          <CalendarIcon />
+          <StaticListItemContent>
+            <StackLayout gap={0.5}>
+              <Text color="inherit">{title}</Text>
+              <Text styleAs="label" color="secondary">
+                {time}
+              </Text>
+            </StackLayout>
+          </StaticListItemContent>
+        </StaticListItem>
+      ))}
+    </StaticList>
+  );
+};
+
+export const WithButtons: StoryFn<StaticListProps> = () => {
+  const id = useId();
+
+  return (
+    <StaticList style={{ width: "320px" }}>
+      {complexEventsData.map(({ title, time }) => (
+        <StaticListItem key={title}>
+          <StaticListItemContent>
+            <StackLayout gap={0.5}>
+              <Text color="inherit" id={id}>
```

Comment:
This will generate duplicate ids? You should just prefix/suffix this with the title